### PR TITLE
Note DynamoDB Local doesn't support transactions

### PR DESCRIPTION
**Pull request by adamchainz**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/DynamoDBLocal.UsageNotes.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ The downloadable version of DynamoDB is intended for development and testing pur
 
 The downloadable version of DynamoDB differs from the web service in the following ways:
 + Regions and distinct AWS accounts are not supported at the client level\.
++ Transactiosn are not supported\.
```
Comment (dehli) [inline doc_source/DynamoDBLocal.UsageNotes.md]: Typo with "Transactions"

Comment (adamchainz) [reply doc_source/DynamoDBLocal.UsageNotes.md]: thansk

**Comment by chris-erickson** [comment]
Would be nice to get this fixed or noted.

**Comment by chris-erickson** [comment]
Or close this PR?? https://aws.amazon.com/about-aws/whats-new/2019/02/amazon-dynamodb-local-adds-support-for-transactional-apis-on-demand-capacity-mode-and-20-gsis/

**Comment by adamchainz** [comment]
📪

**Pull request closed without merging**